libnd4j/include/ops/declarable/generic/parity_ops/bincount.cpp

```diff
@@ -46,6 +46,9 @@ namespace nd4j {
             int maxIndex = in->argMax();
             int maxLength = int((*in)(maxIndex))  + 1;
 
+            if (block.numI() > 0)
+                maxLength = nd4j::math::nd4j_max(maxLength, INT_ARG(0));
```

Comment:
sorry, i dont remember description of this op, but shape fn conflicts with impl fn: minlength vs maxlength

Reply:
Conflicts how?